tests/test_views.py

```diff
@@ -159,6 +169,16 @@ def run_check(self):
         response = client.get(self.url, HTTP_ACCEPT='application/octet-stream')
         assert response['content-type'] == 'text/html; charset=utf-8'
 
+    def test_success_unsupported_and_supported_accept(self, client):
```

Comment:
Same here, I believe this is covered, isn't it?

Reply:
This test was failing with the old code, because the loop was only considering the first element in the sorted list.